### ISSUE TITLE
After importing from a network device, importing more content for the same channel from studio always gives an error

### ISSUE THREAD

**Issue by marcellamaki** [learningequality/kolibri#12300]
## Observed behavior
Noted by @jredrejo during the 0.17.0-alpha0 bug bash
[notion](https://www.notion.so/learningequality/After-importing-from-a-network-device-importing-more-content-for-the-same-channel-from-studio-alway-b092ad1535394a39b4da7a7c45647aee)

## Errors and logs

```
device/#/content:1 Unchecked runtime.lastError: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received
```

## Expected behavior
User should be able to return to the same channel and import more content from studio

## Context
0.17.0-alpha0

**Comment by marcellamaki** [comment]
@radinamatic @pcenov could we also get some follow up testing/details here? :thankyou:

**Comment by pcenov** [comment]
Hi @marcellamaki, I am not able to replicate this one:

https://github.com/learningequality/kolibri/assets/79847249/af924bbd-6547-4ff9-b17f-168008c75bdb

**Comment by jredrejo** [comment]
@pcenov there's one more thing that might make this hard to replicate: I first imported from a network device, but only part of the channel. Then connecting to studio, the channel had a newer version, importing from Studio caused the error everytime I tried. 
After that, I also tried to upgrade the channel in the network device and upgrade from it, but it was impossible too.
The only solution was to delete the imported resources and import from scratch from Studio.

Anyway, I don't think this is a very odd scenario: when being in a network without Internet access, importing a non-upgraded channel from a network device is logical. If you get Internet access later, trying to upgrade it from Studio makes sense
